book_source/models/index.Rmd

```diff
@@ -0,0 +1,26 @@
+#Models
+
+This section will contain information about all models and output variables that are supported by PEcAn.
+
+| Model Name | Available in the VM | Prescribed Inputs | Input Functions/Values | Restart Function |
```

Comment:
Probably would be good to store this in BETY as well? Or in some other database that gets automatically updated.